Dockerfile_verrazzano

```diff
@@ -0,0 +1,40 @@
+FROM container-registry.oracle.com/os/oraclelinux:7.8 as build_base
```

Comment:
need copyright/license headers - and i now realize i might have neglected that on the earlier ones too :(